src/main/java/qupath/ext/template/ui/InterfaceController.java

```diff
@@ -22,6 +20,11 @@ public class InterfaceController extends VBox {
     @FXML
     private Spinner<Integer> threadSpinner;
 
+    /**
+     * Create a new instance of the interface controller.
+     * @return
```

Comment:
Does an empty `@return` do anything? It just gives me an nag message in the IDE without adding any info (I mean, docing the return here is redundant anyhow)

Reply:
It gives a load of complaints when I `./gradlew build` without it.

Reply:
Sadness either way then. Fair enough, this is the autogen so least effort. I'll see if I can suppress the warning globally for my own sanity